public/src/components/amounts/amount.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import update from 'immutability-helper';
+
+import {createStyles, Theme, withStyles, WithStyles} from "@material-ui/core";
+import {Amount} from './amounts';
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import Button from "@material-ui/core/Button";
+
+
+const styles = ({ palette, spacing, mixins }: Theme) => createStyles({
+  amount: {
+    marginLeft: spacing.unit,
+    marginRight: spacing.unit
+  },
+  isDefault: {
+    color: 'red'
+  }
+});
+
+interface Props extends WithStyles<typeof styles> {
+  amount: Amount,
+  deleteAmount: (amountToDelete: string) => void,
+  makeDefault: (defaultAmount: string) => void
+}
+
+interface AmountState {
+  amount: Amount,
+  anchor: HTMLElement | null
```

Comment:
a comment on what the `anchor` is, and why it needs to be tracked in the state, would be helpful